docs/metrics/metrics-queries/metric-query-error-messages.md

```diff
@@ -15,9 +15,9 @@ Sumo Logic imposes limits on the input data for a query and the data output by t
 
 *Input data* is the data that matches the selector, prior to aggregation. Sumo Logic evaluates the volume of input data in terms of the number of time series.
 
-For a single metrics query row, Sumo Logic limits the number of input time series to 1000 for non-aggregate queries, and 37000 for aggregate queries (queries that have an aggregate operator like avg and max).
+For a single metrics query row, Sumo Logic limits the number of input time series to 1000 for non-aggregate queries, and 37000 for aggregate queries (queries that have an aggregate operator like avg and max). In addition, the total number of datapoints scanned by a single row of query limited at 500,000,000 (500M) raw datapoints, or 50M [rollup datapoints](https://help.sumologic.com/docs/metrics/introduction/metric-quantization/#rollup-types ).
```

Comment:
1. Inconsistent formatting: we mention `500,000,000 (500M)` raw DPs but only `50M` rollup DPs.
2. Unnecessary space between `#rollup-types` and `)`.

Reply:
Also, in the referenced document, we write *data points* not *datapoints*.

Reply:
I have made the update. Thanks!